pkg/commands/compute/compute_test.go

```diff
@@ -83,78 +83,54 @@ func getFlags(cmd *kingpin.CmdClause) reflect.Value {
 }
 
 func TestCreatePackageArchive(t *testing.T) {
-	for _, testcase := range []struct {
```

Comment:
The changes to this file can be summarised as follows...

Remove the 'table test' structure as it was redundant (there was only ever one scenario).